CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v0.5.0](https://github.com/puppetlabs/puppetlabs-exec/tree/v0.5.0) (2019-09-05)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-exec/compare/0.4.0...v0.5.0)
+
+### Changed
+
+- pdksync - \(MODULES-8444\) - Raise lower Puppet bound [\#75](https://github.com/puppetlabs/puppetlabs-exec/pull/75) ([david22swan](https://github.com/david22swan))
+
+### Added
+
+- \(FM-8217\) Port module to litmus [\#83](https://github.com/puppetlabs/puppetlabs-exec/pull/83) ([michaeltlombardi](https://github.com/michaeltlombardi))
```

Comment:
Is this an addition or a maint?

Reply:
@lombardi porting to litmus is a feature label hence its addition